lib/openstax/exchange/fake_client/fake_client.rb

```diff
@@ -43,20 +43,18 @@ def token
       end
 
       def create_identifier
```

Comment:
It seems like this method is now mis-named.  Perhaps we should have `#create_read_identifier`, `#create_write_identifier`, and `#create_identifiers`?